ci/actions/dev-build-tag-gen.sh

```diff
@@ -133,9 +133,16 @@ if [[ $previous_release_gen == false ]]; then
     last_tag=$(get_first_item "$version_tags")
 else
     previous_release_major=$(( current_version_major - 1 ))
-    version_tags=$(git tag | sort -V -r | grep -E "^(V(${previous_release_major}).([0-9]+)(DB[0-9]+)?)$" || true)
+    version_tags=$(git tag | sort -V -r | grep -E "^(V(${previous_release_major}).([0-9]+))$" || true)
```

Comment:
This block of code is difficult to read, some comments explaining what is happening would be nice.

Reply:
Sent a new commit adding more comments, let me know if they are good.